buildReleases.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import argparse
+import csv
+import yaml
+import os
+import subprocess
+
+def generateYMLfromCSV(csvFile, ymlFile):
+  print("\n #####  Converting csv at {} to yml at {} #####\n".format(csvFile, ymlFile))
+  with open(csvFile, mode='r',encoding='utf-8') as csv_file:
+    csv_reader = csv.reader(csv_file, delimiter='\t')
+    english = dict()
+    spanish  = dict()
+    for en, es, key in iter(csv_reader):
+      if key:
+        category, subKey = key.split('.')
+        if category not in english:
+          english[category] = dict()
+          spanish [category] = dict()
+        english[category][subKey] = en
+        spanish [category][subKey] = es
+  f = open(ymlFile, 'w',encoding='utf-8')
+  # print(english)
+  f.write(yaml.dump({ "en": english, "es": spanish }, default_flow_style=False, allow_unicode=True))
+
+def buildWebRelease():
+  print("\n #####  Building site_en #####\n")
+  subprocess.run('jekyll build JEKYLL_ENV=es -d ./_site_en/ JEKYLL_ENV=es', shell=True)
```

Comment:
I think you have the commands for English/Spanish reversed?

Reply:
Oh nvm, but you can remove the JEKYLL_ENV=es here. I don't think it does anything.

Reply:
fixed, TY